### DISCUSSION TITLE
What does combat mode do?

### DISCUSSION THREAD

**Discussion by Daza99** [luiscastilho/dungeoneering#15] [Q&A] [answered]
Doesn't appear to have a function? there is no text to say what it does.

**Comment by luiscastilho** [comment, marked as answer]
It's a feature that changed in version 1.3. Once you have added tokens to the initiative (right click on a token > Settings > Toggle initiative), this button shows/hides the initiative bar, where you can drag icons left or right to set initiative order in combat.

I will add some instructions if combat mode is toggled on and initiative is empty. Good idea.
